cirq/sim/wave_function_test.py

```diff
@@ -25,32 +25,62 @@ def assert_dirac_notation(vec, expected, decimals=2):
     assert cirq.dirac_notation(np.array(vec), decimals=decimals) == expected
 
 
-def test_bloch_vector_simple():
+def test_bloch_vector_simple_H_zero():
     sqrt = np.sqrt(0.5)
-    test_state = np.array([sqrt, sqrt])
+    H_state = np.array([sqrt, sqrt])
 
-    bloch = cirq.bloch_vector_from_state_vector(test_state, 0)
+    bloch = cirq.bloch_vector_from_state_vector(H_state, 0)
     desired_simple = np.array([1,0,0])
     np.testing.assert_array_almost_equal(bloch, desired_simple)
 
 
-def test_bloch_vector_invalid():
-    with pytest.raises(ValueError):
-        _ = cirq.bloch_vector_from_state_vector(
-            np.array([0.5, 0.5, 0.5]), 0)
-    with pytest.raises(IndexError):
-        _ = cirq.bloch_vector_from_state_vector(
-            np.array([0.5, 0.5,0.5,0.5]), -1)
-    with pytest.raises(IndexError):
-        _ = cirq.bloch_vector_from_state_vector(
-            np.array([0.5, 0.5,0.5,0.5]), 2)
+def test_bloch_vector_simple_XH_zero():
+    XH_state = np.array([0.70710678, 0.70710678])
```

Comment:
use `np.sqrt(0.5)` instead of the written out value

Reply:
fixed